fvm/evm/stdlib/contract.cdc

```diff
@@ -127,9 +132,10 @@ contract EVM {
     /// Creates a new bridged account
     access(all)
     fun createBridgedAccount(): @BridgedAccount {
```

Comment:
should we rename to createCOA

Reply:
I tried that but it was a big change, I moved it to another PR to rename all (and tests)

Reply:
+1 on renaming, but also doing that it in a separate PR, to keep PRs small